docs/resim/ros2/index.md

```diff
@@ -0,0 +1,92 @@
+# ROS2
+
+The [Robot Operating System](https://www.ros.org/) or ROS and its associated
+libraries represent one of the most ubiquitous collections of robotics software
+in use today. We therefore provide first class support for using ROS2 with
+ReSim's tools. To enable this without making ReSim's tools explicitly depend on
+ROS2 libraries, we provide a converter binary that can be used to convert ROS2
+bags to the [ReSim Log Format](../msg) which our other tools and libraries
+natively consume. 
+
+## Basic Conversion
+
+If you're using common ROS2 message types (e.g. those in
+`std_msgs` or `geometry_msgs`) to log your topics of interest the conversion is
+as simple as cloning [resim/open-core](https://github.com/resim-ai/open-core/)
+and running the following commands.
+
+```bash
+bazel run //resim/ros2:convert_log -- \
+    --log </path/to/my/rosbag_dir/>   \
+    --output </path/to/my/converted/log.mcap>
+```
+
+If there are common public types that you need which we don't yet support, feel
+free to reach out and we can add them.
+
+A pybound version of this converter is
+also provided in the `resim_ros2` package distributed through our open-core
+[Releases](https://github.com/resim-ai/open-core/releases) and can be used like
+so:
+
+```python
+import resim.ros2.resim_log_from_ros2_python as rlr2
+from resim.ros2 import RESIM_DIR
+
+# ...
+
+rlr2.resim_log_from_ros2(
+    str(RESIM_DIR / "ros2" / "default_converter_plugin.so"), # See plugin info below
+    input_log_path,
+    converted_log_path,
+```
```

Comment:
(nit) I think an extra new line would help the syntax highlighter here!

Reply:
I'm kind of unable to get the syntax highlighter working under any circumstances. 😕 
I've been tweaking the mkdocs.yml in our docs repository a bit, and I can only get highlighting which does not work in dark mode (the text is unreadable). I'll leave this as is for now.